apps/contrib/templatetags/item_tags.py

```diff
@@ -0,0 +1,66 @@
+from django import template
+from django.core.urlresolvers import reverse
+
+register = template.Library()
+
+
+@register.assignment_tag
+def get_item_view_permission(item):
+    return get_item_permission(item, 'view')
+
+
+@register.assignment_tag
+def get_item_add_permission(item):
+    return get_item_permission(item, 'add')
+
+
+@register.assignment_tag
+def get_item_change_permission(item):
+    return get_item_permission(item, 'change')
+
+
+@register.assignment_tag
+def get_item_delete_permission(item):
+    return get_item_permission(item, 'delete')
+
+
+@register.assignment_tag
+def get_item_permission(item, verb):
+    return '{app}.{verb}_{name}'.format(
+        app=item._meta.app_label,
+        verb=verb,
+        name=item._meta.verbose_name
+    )
+
+
+@register.assignment_tag
+def get_item_detail_url(item):
+    return get_item_url(item, 'detail')
+
+
+@register.assignment_tag
+def get_item_create_url(item):
+    return get_item_url(item, 'create')
+
+
+@register.assignment_tag
+def get_item_update_url(item):
+    return get_item_url(item, 'update')
+
+
+@register.assignment_tag
+def get_item_delete_url(item):
+    return get_item_url(item, 'delete')
```

Comment:
Why are there all these additional tags. We can just use `{% get_item_url object 'delete' as delete_url %}`, right?